connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufFunctionsSuite.scala

```diff
@@ -26,11 +26,11 @@ import com.google.protobuf.{ByteString, DynamicMessage}
 import org.apache.spark.sql.{Column, QueryTest, Row}
 import org.apache.spark.sql.AnalysisException
 import org.apache.spark.sql.functions.{lit, struct}
-import org.apache.spark.sql.protobuf.protos.SimpleMessageProtos.SimpleMessageRepeated
+import org.apache.spark.sql.protobuf.protos.SimpleMessageProtos.{EventRecursiveA, EventRecursiveB, OneOfEvent, OneOfEventWithRecursion, SimpleMessageRepeated}
```

Comment:
Are there tests for recursive fields?

Reply:
@rangadi yes, 
`Handle recursive fields in Protobuf schema, C->D->Array(C)` and
`Handle recursive fields in Protobuf schema, A->B->A`

Reply:
Could we move that to different tests?

Reply:
@rangadi I didn't understand. these are already two different tests.